### PR TITLE
feature:allow set vars filter in router configuration page

### PR DESCRIPTION
**Pull request by dingchaofa**
feature: https://github.com/apache/incubator-apisix-dashboard/issues/161

### CODE REVIEW

File: vue.config.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ module.exports = {
   productionSourceMap: false,
   devServer: {
     // https://cli.vuejs.org/zh/config/#devserver-proxy
-    proxy: 'http://127.0.0.1:9080/apisix/admin/'
+    proxy: 'http://192.168.10.152/apisix/admin/'
```
Comment (juzhiyuan) [inline vue.config.js]: Please use localhost

File: src/views/schema/routes/edit.vue [hunk 2]
```diff
@@ -497,5 +511,15 @@ export default class extends Vue {
       }
     }
   }
+  .var-item {
+    .el-form-item {
+      margin-bottom: 10px;
+      display: inline-block;
+      .el-form-item__label {}
```
Comment (juzhiyuan) [inline src/views/schema/routes/edit.vue]: Please remove this line

File: src/components/VarArgs/index.vue [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    (this.vars as any).push({
+      name: null,
+      operator: null,
+      value: null
+    })
+    this.onChange()
+  }
+
+  private removeVar(index:number) {
+    this.$confirm(`Do you want to remove the var?`, 'Warning', {
```
Comment (juzhiyuan) [inline src/components/VarArgs/index.vue]: Please use `''` when there has no variables

File: vue.config.js (hunk 1 above)
Comment (dingchaofa) [reply vue.config.js]: fixed

File: src/views/schema/routes/edit.vue (hunk 2 above)
Comment (dingchaofa) [reply src/views/schema/routes/edit.vue]: fixed and other file also was fixed.

File: src/components/VarArgs/index.vue (hunk 3 above)
Comment (dingchaofa) [reply src/components/VarArgs/index.vue]: fixed and other file also was fixed.

**Pull request merged**